data/coalition/coalition missions.txt

```diff
@@ -1866,3 +1866,254 @@ mission "Saryd University Lecture"
+			`	"My ancestors," the short one interrupts him.`
+			`	"What our ancestors created is."`
+			`	They show you a device with a map on screen, depicting nearby star systems, and the short Arach starts explaining, "Intrigued as to just how accurate its predictions have become, we are, so to verify its data about a star deeper in human space, the next step is." He points to a system at the edge of the map. "Have humans discovered this system, Captain? Interested in comparing the probe's data on far away binary stars, the president is."`
+			`	You look at the screen and at your own map for a moment, and recognize the system as <waypoints>. "Settled it is, then. Head to that system, you should," the taller one speaks again. "Still with our adjustments, your scanners are, so just tinker them a bit to gather the needed, we will."`
+			`	A few minutes later, they're done with your scanners, and leave your ship, wishing you good luck.`
+				accept
+	on enter "Alnasl"
+		dialog `One of your monitors brings up various sets of data written in the Arach language when you enter the system, and your scanners begin gathering the data House Idriss wants. A few minutes pass, and just when it seems to end, a loading bar comes up. When it reaches the end, another one appears, and the process repeats a few more times. After about an hour, your monitor goes back to normal.`
+	on complete
+		event "outskirts gauger available" 25 63
+		payment 250000
+		conversation
+			`Flopug is waiting by the landing bays for you, with the Arach duo beside her. They come inside your ship when you've landed, and start checking the data, after Flopug pays you <payment>.`
+			`	"Umm... more errors with this one, the probe committed," the taller Arach says. "Gotten a good deal right, it still did, but a shame it is, that deal with that much more of a distance, it couldn't."`
+			`	"Not distance, the cause of errors was," the short Arach responds. "Confused with the binaries' signatures, it got. Refined to account for binary star systems and their oddities, its algorithms had to be."`
+			`	"At any rate, happy to call the probe's voyage a success, I am," Flopug cuts them off. "Expected it to have developed this well, we hadn't, when notified of its existence by... I mean, when remembered about the approaching date of its arrival to that system, we did."`
+			`	They turn you scanners back to normal, and tell you they'll work on using the data to build a new scanner, and will let you know about it when it's ready, the next time you're on <origin>. They thank you for your help again, and leave your ship.`
+
+event "outskirts gauger available"
+	planet "Ablub's Invention"
+		outfitter "Outskirts Gauger"
+
+mission "Coalition: Outskirts Gauger Ready"
+	landing
+	invisible
+	to offer
```

Comment:
Perhaps this should have some sort of timer? As it stands it's possible for this mission to trigger the very next day, which is probably not in line with designing and building a new scanner.

Reply:
There's `event "outskirts gauger available" 25 63` on line 2094, is that not working properly? I thought that's how you did it to have a range for the timer, this one being from 25 to 63 days.

Reply:
Oops, I completely missed that. Feel free to resolve.